Chaincase.UI/wwwroot/style/global.css

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
would be good to test if styles from here are applied 2

Reply:
Did some local testing. Didn't have any actual styles to apply at the time